### DISCUSSION TITLE
ScriptReference/XR.Hand

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#520] [Notes]
# ScriptReference/XR.Hand

https://docs.unity3d.com/ScriptReference/XR.Hand.html

**Comment by Fangh** [comment]
There is an issue with it : https://forum.unity.com/threads/hand-tracking-with-xr-plug-in-management-oculus-plugin-openxr-plugin.1111639/#post-7217605
